packages/schema/parse/src/comparison/comparers/generic-type.ts

```diff
@@ -0,0 +1,62 @@
+import { GenericDefinition } from "../../typeInfo";
+import { CompareOptions, CompareResult, VersionRelease } from "./utils";
+
+export function compareGenericType(
+  obj1: GenericDefinition,
+  obj2: GenericDefinition,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
since it's not used you can just remove it

Reply:
All the other compare functions have the same signature so I was just reluctant to have a different one for just this one.